### PR TITLE
fix(raft-exp-yaml): rename raft experimetal feature flag

### PR DESCRIPTION
**Pull request by aleksbykov**
new raft experimental feature flag name is consistent-topology-changes.

### Testing
- [longevity-schema-topology-changes-12h-with-raft-test](https://jenkins.scylladb.com/job/scylla-master/job/raft/job/longevity-schema-topology-changes-12h-with-raft-test/3/console) - Passed
- [longevity-mv-si-4days-test](https://jenkins.scylladb.com/job/scylla-master/job/raft/job/longevity-mv-si-4days-test/3/) - Failed by reason not related to fix
- [longevity-100gb-4h-with-raft-test](https://jenkins.scylladb.com/job/scylla-master/job/raft/job/longevity-100gb-4h-with-raft-test/17) - Passed
### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [ ] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevent to this change (if needed)

### CODE REVIEW

File: sdcm/utils/raft/__init__.py [hunk 1]
```diff
@@ -315,16 +315,16 @@ def is_cluster_topology_consistent(self) -> bool:
         return len(token_ring_ids) == num_of_nodes
 
 
-def get_raft_mode(node) -> Raft | NoRaft:
+def get_raft_mode(node) -> RaftFeature | NoRaft:
     with node.remote_scylla_yaml() as scylla_yaml:
         if node.is_kubernetes():
             consistent_cluster_management = scylla_yaml.get('consistent_cluster_management')
         else:
             consistent_cluster_management = scylla_yaml.consistent_cluster_management
```
Comment (fruch) [inline sdcm/utils/raft/__init__.py:323]: is this configuration option still exist ?

**Comment by aleksbykov** [comment]
@fruch  @soyacz @temichus can you please review

**Comment by fruch** [comment]
> @fruch  @soyacz @temichus can you please review

Just to clarify, the direction you are taking is to reuse all the raft jobs ?

Please go over the jobs, and review, cause I think I've deleted few of those folder from master

**Comment by aleksbykov** [comment]
> > @fruch  @soyacz @temichus can you please review
> 
> Just to clarify, the direction you are taking is to reuse all the raft jobs ?

Yes, i want to rerun jobs with raft topology feature while it is experimental, to validate that sct will not be broken when raft topology will be moved to GA.

> 
> Please go over the jobs, and review, cause I think I've deleted few of those folder from master
I checked and most of required jobs are under jenkins-pipelines/oss/raft and still available on master



> Just to clarify, the direction you are taking is to reuse all the raft jobs ?

Yes, t

**Review by soyacz — APPROVED** [review]
Assuming this is temporary solution as soon it will be out of experimental - LGTM

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**